ui/components/ui/box/README.mdx

```diff
@@ -107,18 +107,10 @@ GREY: 'grey',
 NEUTRAL_GREY: 'neutral-grey',
 WHITE: 'white',
 PRIMARY1: 'primary-1',
-PRIMARY2: 'primary-2',
 PRIMARY3: 'primary-3',
 SECONDARY1: 'secondary-1',
-SECONDARY2: 'secondary-2',
-SECONDARY3: 'secondary-3',
 SUCCESS1: 'success-1',
-SUCCESS2: 'success-2',
 SUCCESS3: 'success-3',
 ERROR1: 'error-1',
-ERROR2: 'error-2',
-ERROR3: 'error-3',
 ALERT1: 'alert-1',
-ALERT2: 'alert-2',
-ALERT3: 'alert-3',
```

Comment:
Do these remove colors that aren't used at all?

Reply:
Yep; I've done searches and some of these aren't even used.